src/core_landice/Registry.xml

```diff
@@ -1304,6 +1390,46 @@ is the value of that variable from the *previous* time level!
 	       description="generic work array with dimensions of (nCells)"
 	       persistence="scratch"
 	       />
+       <var name="meanFlowParamAVar" type="real" dimensions="nCells" units="Pa^{-3} s^{-1}"
```

Comment:
Remove "Var" from each of these names.  E.g., change `meanFlowParamAVar` to `meanFlowParamA`.

Reply:
Removed